stock_available_to_promise_release/models/stock_move.py

```diff
@@ -591,6 +597,39 @@ def _get_chained_moves_iterator(self, chain_field):
             yield moves
             moves = moves.mapped(chain_field)
 
+    def _return_quantity_in_stock(self, quantity_to_return):
+        picking = self.picking_id
+        picking.ensure_one()
+
+        returned_quantity = 0
+        # create return
+        return_type = picking.picking_type_id.return_picking_type_id
+        wiz_values = {
+            "picking_id": picking.id,
+            "original_location_id": picking.location_dest_id.id,
```

Comment:
This is not necessary. Otherwise, it should be
```suggestion
            "original_location_id": picking.location_id.id,
```

Reply:
@mmequignon Can you drop this line?